### ISSUE TITLE
does not produce any files or diagnostic output 

### ISSUE THREAD

**Issue by zjttoefs** [nuex/wiki_extract#5]
The script just quickly and quietly returns. I'm not a ruby person, so no more in depth analysis right now...

**Comment by nuex** [comment]
Yeah, I don't think this was ever completed. The project should probably be removed.
